T2CC_APP/t2cc/app/src/main/res/layout/activity_browseclass.xml

```diff
@@ -89,20 +89,29 @@
 
         <TextView
             android:id="@+id/classNameLabel"
-            android:layout_width="180dp"
+            android:layout_width="120dp"
             android:layout_height="wrap_content"
             android:layout_gravity="bottom"
             android:layout_weight="0"
             android:text="Class Name"
             android:textStyle="bold" />
 
+        <TextView
+            android:id="@+id/browseTeacherEmailLabel"
+            android:layout_width="65dp"
+            android:layout_gravity="bottom"
+            android:layout_height="wrap_content"
+            android:textStyle="bold"
+            android:layout_weight="0"
+            android:text="Email" />
```

Comment:
I though we changed it to email since email was unique and only one can exists for each account? Either or, I can changed it if you already have the backend setup.